docs/source/Intro.rst

```diff
@@ -24,6 +24,13 @@ Challenges for accurate weather predictions still exist in numerical models. To
     - Larger TC False Alarm Rate in the western North Atlantic
     - Exacerbated low instability bias, which is largely related to dry soil moisture
     - Poor representation of radiation inversions
+  .. group-tab:: RRFS.v1alpha
```

Comment:
Thank you Linlin for adding the SRW component. 
I think RRFS.v1alpah needs some background introduction in the Intro.rst. Meanwhile, is it possible to find a reference link for the known biases for the RRFS?

Reply:
Yes we can add some background introduction, and we can add a reference link for the known biases when it is available.